modules/social_features/social_event/social_event.module

```diff
@@ -638,3 +638,48 @@ function social_event_get_current_event() {
 
   return NULL;
 }
+
+/**
+ * Implements hook_preprocess_field().
+ */
+function social_event_preprocess_field(&$variables) {
+  $formatter = $variables['element']['#formatter'];
+  if (in_array($formatter, ['address_plain', 'address_default'])) {
+    $entity = $variables['element']['#object'];
+    if ($entity && $entity instanceof NodeInterface && $entity->getType() === 'event') {
+      $social_event_settings = \Drupal::config('social_event.settings');
+      $address_visibility_settings = $social_event_settings->get('address_visibility_settings');
+      if (isset($address_visibility_settings['street_code_private']) && !empty($address_visibility_settings['street_code_private'])) {
+        $enrollments = \Drupal::entityTypeManager()->getStorage('event_enrollment')->loadByProperties([
```

Comment:
@denysyukm I see you created helper function _social_geolocation_maps_get_enrollment_status - maybe it make sense to move it here?

Reply:
Good idea! Done